packages/nimbus/src/components/group/group.recipe.ts

```diff
@@ -0,0 +1,14 @@
+import { defineRecipe } from "@chakra-ui/react";
+
+/**
+ * Recipe configuration for the Group component.
+ * Defines the styling variants and base styles using Chakra UI's recipe system.
+ */
+export const groupRecipe = defineRecipe({
```

Comment:
Instead of making the group a compound child of `Toolbar`, I instead defined it as a separate component here, with the idea that it can be fleshed out later if necessary